tests/utils/TestHelper.ts

```diff
@@ -18,7 +19,13 @@ type MockFetch = ReturnType<typeof jest.fn> & {
     resume?: () => Promise<void>;
 };
 
-type Response = {ok: boolean; json: () => Promise<{jsonCode: number}>};
+type Response = {
```

Comment:
Could we use `Response` from `src/types/onyx/Response.ts`?

Reply:
Thank you! Got it 👍 
Updated